cmd/aws-cloud-controller-manager/main.go

```diff
@@ -107,28 +109,74 @@ func main() {
 				}
 			}
 
-			printFlags(cmd.Flags())
+			cliflag.PrintFlags(cmd.Flags())
 
-			c, err := s.Config(KnownControllers(), ControllersDisabledByDefault.List())
+			c, err := s.Config(KnownControllers(), app.ControllersDisabledByDefault.List())
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "%v\n", err)
 				os.Exit(1)
 			}
 
-			if err := app.Run(c.Complete(), wait.NeverStop); err != nil {
+			// initialize cloud provider with the cloud provider name and config file provided
+			cloud, err := cloudprovider.InitCloudProvider(cloudProvider, "")
```

Comment:
I think this is supposed to pass in the cloud provider name, which is stored in the `cloudProvider` variable above.

Reply:
gotcha, will update!